Headers/AppKit/NSTextView.h

```diff
@@ -247,6 +248,9 @@ therefore be stored in the NSLayoutManager to avoid problems.
   // Text checking (spelling/grammar)
   NSTimer *_textCheckingTimer;
   NSRect _lastCheckedRect;
+
+  // NSTextFinder vars...
+  NSArray *_selectedRanges;
```

Comment:
You are currently not using this new instance variable. Either implement something based on this or remove the declaration again.

Reply:
Removed.